.github/workflows/mdet.yml

```diff
@@ -52,7 +52,8 @@ jobs:
       - name: test
         shell: bash -l {0}
         run: |
-          pip install git+https://github.com/esheldon/ngmix.git@master
+          # testing on ngmix2
+          pip install git+https://github.com/esheldon/ngmix.git@api-proposal
           pushd mdet_tests
```

Comment:
These lines and the ones below are why we need to shim the APIs. I forgot about this. I am doing a regression test against v1.3.8.

Reply:
I'm just doing this temporarily so I don't  get emails saying the tests failed

Reply:
This test won't run at all unless the APIs are shimmed in mdet.

Reply:
I misunderstood what was going on there; looks like you compare to a file produced with certain versions, whereas I thought you had been comparing to a fixed file

Reply:
Yep. We didn't want the file in the repo. Then we merged it by accident anyway due to my typo. 🤦

Reply:
Once 2.0.0 is released and has passed against 1.3.8, we can drop support in mdet for 1.3.8.